dll/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ install(DIRECTORY "${LASZIP_HEADERS_DIR}"
 )
 
 install(DIRECTORY "${CMAKE_BINARY_DIR}/include/laszip/"
-        DESTINATION "${LASZIP_INCLUDE_INSTALL_DIR}"
+        DESTINATION "${LASZIP_INCLUDE_INSTALL_DIR}/laszip"
```

Comment:
There's situations where this ends up getting installed in `PREFIX/include/laszip/laszip`, which caused me to start messing with this stuff in the first place.